### PR TITLE
Chore/remove unused

### PR DESCRIPTION
**Pull request by tomchentw**
## Depends on

~#30~

## Description

* Remove several unused/unreachable code

### CODE REVIEW

File: src/components/ContextMenu/index.js [hunk 1]
```diff
@@ -70,9 +70,9 @@ const enhance = compose(
       event.preventDefault();
       const { CmdHandler } = pttchrome;
       const doDOMMouseScroll =
-        CmdHandler.getAttribute("doDOMMouseScroll") === "1";
+        CmdHandler.getAttribute("USING_doDOMMouseScroll") === "1";
```
Comment (robertabcd) [inline src/components/ContextMenu/index.js]: Why are these all prefixed with `USING_`?

Comment (tomchentw) [reply src/components/ContextMenu/index.js]: I have to prefix some of them with `USING_` so that I can identify unused ones. Should I rename them back?

Comment (robertabcd) [reply src/components/ContextMenu/index.js]: Yes, please do. (since the purpose of the rename is just for tracing code)

**Comment by tomchentw** [comment]
Updated and rebased.

**Pull request merged**